tests/integration_tests/test_keras.py

```diff
@@ -22,16 +23,19 @@ def objective(trial):
             np.zeros((16, 20), np.float32),
             np.zeros((16,), np.int32),
             batch_size=1,
-            epochs=1,
-            callbacks=[KerasPruningCallback(trial, "accuracy")],
+            epochs=epochs,
+            callbacks=[KerasPruningCallback(trial, "accuracy", interval=interval)],
             verbose=0,
         )
 
         return 1.0
 
     study = optuna.create_study(pruner=DeterministicPruner(True))
     study.optimize(objective, n_trials=1)
-    assert study.trials[0].state == optuna.trial.TrialState.PRUNED
+    if interval <= epochs:
+        assert study.trials[0].state == optuna.trial.TrialState.PRUNED
+    else:
+        assert study.trials[0].state == optuna.trial.TrialState.COMPLETE
```

Comment:
Just a nit, and we can fix it in another follow up PR.

How about adding this line below?
```
assert study.trials[0].value == 1.0
```

Reply:
Right, I'll open a follow up PR once this PR is merged.